.gitignore

```diff
@@ -1,9 +1,11 @@
 .DS_STORE
 dump.rdb
 target/
+*.log
+doc_data.txt
+
 node_modules/*
 !node_modules/oae-*
-node_modules/oae-rest
+!node_modules/oae-*/*
```

Comment:
Can you add a comment as it not immediately obvious and an oaf like me might remove it when trying to micro optimize?